src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/base_js.html.twig

```diff
@@ -67,6 +67,14 @@
                 localStorage.setItem(profilerStorageKey + name, value);
             },
 
+            addEvent = function (element, eventName, callback) {
+                if (document.all) {
```

Comment:
@aik099  could you do this check around the function definition instead of doing it at runtime to do it only once ? And checking for ``document.addEventListener`` would indeed be better than ``document.all``